### PR TITLE
Add report objects for manageability suite tests

### PR DESCRIPTION
**Pull request by sebrandon1**
build-depends: 28703
- Add compliant/nonCompliant objects for these two tests.
- Add Container.IsTagEmpty() func with corresponding unit tests.

### CODE REVIEW

File: cnf-certification-test/manageability/suite.go [hunk 1]
```diff
@@ -52,15 +53,18 @@ var _ = ginkgo.Describe(common.ManageabilityTestKey, func() {
...
 func testContainersImageTag(env *provider.TestEnvironment) {
-	badContainers := []string{}
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
 	for _, cut := range env.Containers {
 		logrus.Debugln("check container ", cut.String(), " image should be tagged ")
-		if cut.ContainerImageIdentifier.Tag == "" {
-			badContainers = append(badContainers, cut.String())
+		if cut.IsTagEmpty() {
+			nonCompliantObjects = append(nonCompliantObjects, testhelper.NewContainerReportObject(cut.Namespace, cut.Podname, cut.Name, fmt.Sprintf("Container %s is missing image tag(s)", cut.String()), false))
```
Comment (edcdavid) [inline cnf-certification-test/manageability/suite.go:61]: no need to add the container id -> fmt.Sprintf("Container %s is missing image tag(s)", cut.String()) -> "Container is missing image tag(s)"

File: cnf-certification-test/manageability/suite.go [hunk 2]
```diff
@@ -76,18 +80,17 @@ func testContainerPortNameFormat(env *provider.TestEnvironment) {
...
 	}
-	badContainers := []string{}
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
 	for _, cut := range env.Containers {
 		for _, port := range cut.Ports {
 			if !containerPortNameFormatCheck(port.Name) {
-				badContainers = append(badContainers, cut.String())
 				tnf.ClaimFilePrintf("%s: ContainerPort %s does not follow the partner naming conventions", cut, port.Name)
+				nonCompliantObjects = append(nonCompliantObjects, testhelper.NewContainerReportObject(cut.Namespace, cut.Podname, cut.Name, fmt.Sprintf("ContainerPort %s does not follow the partner naming conventions", port.Name), false))
```
Comment (edcdavid) [inline cnf-certification-test/manageability/suite.go:89]: optional: consider re-using port/protocol fields defined in https://github.com/test-network-function/cnf-certification-test/pull/1207 and add 2 level report, one per container per port and one per pod

**Review by jmontesi — APPROVED** [review]
Approved waiting for updates after David's comments.

File: cnf-certification-test/manageability/suite.go (hunk 2 above)
Comment (sebrandon1) [reply cnf-certification-test/manageability/suite.go:89]: I'll adjust this once the #1207 gets reviewed and merged.

File: cnf-certification-test/manageability/suite.go (hunk 1 above)
Comment (sebrandon1) [reply cnf-certification-test/manageability/suite.go]: Fixed.

**Review by greyerof — APPROVED** [review]

**Pull request merged**